seed/energy/pm_energy_template/post_process.py

```diff
@@ -0,0 +1,57 @@
+import glob
+import json
+import logging
+import re
+
+_log = logging.getLogger(__name__)
+
+def ns2s(string):
+    '''
+    Convert unit of time from 'ns' to 's'
+    '''
+
+    return string.replace('000000000,', ',')
+
+def post_process(file_path):
+    json_file_path = file_path[len(file_path):-5] + '_json.txt'
```

Comment:
I don't think this line works anyway... the files are all named to _json.txt. file_path is probably an empty string most of the time.